developer-docs-site/package.json

```diff
@@ -23,6 +23,7 @@
     "@docusaurus/plugin-google-gtag": "2.1.0",
     "@docusaurus/preset-classic": "2.1.0",
     "@mdx-js/react": "1.6.22",
+    "aptos": "^1.3.13",
```

Comment:
Indeed, not needed

Reply:
This got inadvertently added when I did the `npm -i aptos` to test the SDK install step. Is there any harm in keeping it this way? Curious.

Reply:
No harm, but there is no reason to include it here, since the docs don't require the `aptos` package. Best not to keep it here.

Reply:
After installing it you'll want to run `yarn install` to fix up the lockfile btw.

Reply:
I had to run first `yarn remove aptos`  followed by `yarn install` (this second step didn't introduce any change though) to remove aptos references.